core/src/main/java/org/jahia/services/importexport/ImportExportBaseService.java

```diff
@@ -2023,7 +2023,7 @@ private String validateZipName(String filename) throws java.io.IOException {
         String canonicalPath = new File(filename).getCanonicalPath();
         String canonicalID = new File(".").getCanonicalPath();
 
-        if (canonicalPath.startsWith(canonicalID)) {
+        if (new File(filename).getCanonicalFile().toPath().startsWith(canonicalID)) {
```

Comment:
This case here is a false positive as it is only for comparing zip file names and is not really reading or writing paths on disk.